pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/signed-exchange.md

```diff
@@ -108,111 +107,69 @@ Ea8/B6hPatJ0ES8q/HO3X8IVQwVs1n3aAr0im0/T+Xc=
-
-
-## Deploy packager to a staging server
-
-You should set Signed Exchanges up on a staging server first, to verify a correct setup, before migrating to production to ensure proper implementation. We recommend using [`amppackager`](https://github.com/ampproject/amppackager/blob/master/README.md) and its built-in web server. 
-
-Follow instructions [here to set up `amppackager` for your site](https://github.com/ampproject/amppackager/blob/master/README.md). 
+Follow instructions [here to set up `amppackager` for your site](https://github.com/ampproject/amppackager/blob/master/README.md).
 
 [tip type="read-on"]
-The [`packager.js`](https://github.com/ampproject/docs/blob/future/platform/lib/routers/packager.js) file for `amp.dev` demonstrates Signed Exchange support setup.
+See [`packager.js`](https://github.com/ampproject/docs/blob/future/platform/lib/routers/packager.js) (used by `amp.dev`) for an example of the server-side changes you will need to make to route the required requests `amppkg`.
 [/tip]
 
-NOTE: Alternatively, you can use the command-line clients [`transform`](https://github.com/ampproject/amppackager/blob/master/transformer/README.md) and [`gen-signedexchange`](https://github.com/WICG/webpackage/tree/master/go/signedexchange), and handle version negotiation and certificate management tasks by hand. This should only be implemented if `amppackager` does not fit for your production environment. You will need to serve with the appropriate content-types (`application/signed-exchange;v=b3` and `application/cert-chain+cbor`). 
+### Testing
 
-Test that your staging site returns the content with the added MIME type `application/signed-exchange` when provided with the correct request headers. In the below example, replace `staging.example.com` with your staging server.
+Verify that your staging site returns the content with the added MIME type `application/signed-exchange` when provided with the correct request headers. In the below example, replace `staging.example.com` with your staging server.
 
-
-```html
-$ curl -si -H 'amp-cache-transform: google' -H 'accept: application/signed-exchange;v=b3;q=0.9,*/*;q=0.8' https://staging.example.com/ | less
+```sh
+$ curl -si -H 'amp-cache-transform: google' -H 'accept: application/signed-exchange;v=b3;q=0.9,*/*;q=0.8' https://staging.example.com/
 ```
 
 This command should return the following:
 
-```html
+```txt
 content-type: application/signed-exchange;v=b3
 ```
 
 [tip type="important"]
 The `v=b3` version string is the current version. This version will change. 
 [/tip]
 
-The response will be a binary encoding of your AMP page.
-
-You can also test with the accept header Chrome sends on navigations:
+The bulk of the response should be your AMP page (in plaintext) together with some binary headers and footers.
```

Comment:
No footers. There's some binary sprinkled in the middle if your page is >16kb.

Reply:
Thanks.